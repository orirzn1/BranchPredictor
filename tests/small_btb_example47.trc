2 1 3 0 global_history global_tables using_share_lsb
0x56e4 N 0x1934
0xbacc T 0x48d58
0xa322648 N 0x7ab28
0x6c4 N 0x816510
0x56e4 N 0x1934
0xbacc N 0x48d58
0x6c4 T 0x816510
0x56e4 T 0x1934
0x56e4 T 0x1934
0x56e4 T 0x1934
0xa322648 N 0x7ab28
0x56e4 T 0x1934
0x56e4 T 0x1934
0x362fc T 0xf60
0xbacc N 0x48d58
0xbacc N 0x48d58
0xbacc N 0x48d58
0x6c4 N 0x816510
0xbacc T 0x48d58
0x6c4 N 0x816510
0xbacc T 0x48d58
0x362fc N 0xf60
0xbacc N 0x48d58
0x6c4 T 0x816510
0x56e4 T 0x1934
0xbacc N 0x48d58
0xa322648 T 0x7ab28
0xbacc N 0x48d58
0x362fc T 0xf60
0x362fc T 0xf60
0x6c4 N 0x816510
0xbacc N 0x48d58
0x56e4 N 0x1934
0x362fc T 0xf60
0x56e4 N 0x1934
0x56e4 T 0x1934
0x56e4 T 0x1934
0x6c4 N 0x816510
0xa322648 T 0x7ab28
0xa322648 T 0x7ab28
0xbacc T 0x48d58
0xbacc N 0x48d58
0xa322648 T 0x7ab28
0xbacc N 0x48d58
0x56e4 T 0x1934
0x362fc T 0xf60