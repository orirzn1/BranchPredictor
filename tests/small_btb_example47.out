0x56e4 N 0x56e8
0xbacc N 0xbad0
0xa322648 N 0xa32264c
0x6c4 N 0x6c8
0x56e4 N 0x56e8
0xbacc N 0xbad0
0x6c4 N 0x6c8
0x56e4 N 0x56e8
0x56e4 N 0x56e8
0x56e4 T 0x1934
0xa322648 N 0xa32264c
0x56e4 N 0x56e8
0x56e4 T 0x1934
0x362fc N 0x36300
0xbacc N 0xbad0
0xbacc N 0xbad0
0xbacc N 0xbad0
0x6c4 N 0x6c8
0xbacc N 0xbad0
0x6c4 N 0x6c8
0xbacc N 0xbad0
0x362fc N 0x36300
0xbacc N 0xbad0
0x6c4 N 0x6c8
0x56e4 N 0x56e8
0xbacc N 0xbad0
0xa322648 N 0xa32264c
0xbacc N 0xbad0
0x362fc N 0x36300
0x362fc N 0x36300
0x6c4 N 0x6c8
0xbacc N 0xbad0
0x56e4 N 0x56e8
0x362fc N 0x36300
0x56e4 N 0x56e8
0x56e4 T 0x1934
0x56e4 N 0x56e8
0x6c4 N 0x6c8
0xa322648 N 0xa32264c
0xa322648 T 0x7ab28
0xbacc N 0xbad0
0xbacc T 0x48d58
0xa322648 N 0xa32264c
0xbacc T 0x48d58
0x56e4 N 0x56e8
0x362fc N 0x36300
flush_num: 22, br_num: 46, size: 73b
